app/scripts/models/app-model.js

```diff
@@ -645,7 +645,7 @@ const AppModel = Backbone.Model.extend({
             this.scheduleBackupFile(file, data);
         }
         if (params) {
-            this.saveFileFingerprint(file, params.password);
+            this.saveFileFingerprint(file, params.password.getText());
```

Comment:
`getText` exposes the password as plaintext in memory. It should be called in the fingerprint handling code and passed from here as `password`.

Reply:
In the fingerprint module in js, closer to the place where it’s used, should be enough. Now it will happen for everyone and too early.

Reply:
I will take a look

Reply:
The place where I suggest moving them is `launcher-cordova.js::Launcher.fingerprints.register`.

Reply:
Done.